README.md

```diff
@@ -207,7 +207,9 @@ Then add the AirGoogleMaps directory:
 
 https://github.com/airbnb/react-native-maps/blob/1e71a21f39e7b88554852951f773c731c94680c9/docs/installation.md#ios
 
-An unofficial step-by-step guide is also available at https://gist.github.com/heron2014/e60fa003e9b117ce80d56bb1d5bfe9e0
```

Comment:
It appears the discussion thread around this gist is still active. I think it's a good idea to leave this here and add your article.

Reply:
Thank you for your return. True, I left it and add there was an active discussion thread on it + add the step by step article and some details. Let me know if you have any other recommandation ! :)